src/Formats/ProtobufSerializer.cpp

```diff
@@ -2648,13 +2750,19 @@ namespace
             boost::range::set_difference(collections::range(column_names.size()), used_column_indices_sorted,
                                          std::back_inserter(missing_column_indices));
 
-            *root_serializer_ptr = message_serializer.get();
-
-#if 0
-            LOG_INFO(&Poco::Logger::get("ProtobufSerializer"), "Serialization tree:\n{}", get_root_desc_function(0));
-#endif
-
-            return message_serializer;
+            if (!with_envelope)
+            {
+                *root_serializer_ptr = message_serializer.get();
+                LOG_INFO(&Poco::Logger::get("ProtobufSerializer"), "Serialization tree:\n{}", get_root_desc_function(0));
```

Comment:
This log was under `#if 0` (because it's used only for debugging). And now there are some failed tests in CI because of this unexpected output.

Reply:
You are right ... I should have ifdef'd that away before pushing. Fixed.